lms/djangoapps/grades/api/serializers.py

```diff
@@ -25,6 +25,58 @@ def to_representation(self, obj):
         # DRF v3 unhelpfully raises an exception.
         return dict(
             super(GradingPolicySerializer, self).to_representation(
-                defaultdict(lambda: None, obj)
+                defaultdict(lambda: None, instance)
             )
         )
+
+
+class SectionBreakdownSerializer(serializers.Serializer):
+    """
+    Serializer for the `section_breakdown` portion of a gradebook entry.
+    """
+    are_grades_published = serializers.BooleanField()
+    auto_grade = serializers.BooleanField()
+    category = serializers.CharField()
+    chapter_name = serializers.CharField()
+    comment = serializers.CharField()
+    detail = serializers.CharField()
+    displayed_value = serializers.CharField()
+    is_graded = serializers.BooleanField()
+    grade_description = serializers.CharField()
+    is_ag = serializers.BooleanField()
+    is_average = serializers.BooleanField()
+    is_manually_graded = serializers.BooleanField()
+    label = serializers.CharField()
+    letter_grade = serializers.CharField()
+    module_id = serializers.CharField()
+    percent = serializers.FloatField()
+    score_earned = serializers.FloatField()
+    score_possible = serializers.FloatField()
+    section_block_id = serializers.CharField()
+    subsection_name = serializers.CharField()
+
+
+class SimpleSerializer(serializers.BaseSerializer):
+    """
+    A Serializer intended to take a dictionary of data and simply spit
+    that same dictionary back out as the "serialization".
+    """
+    def to_representation(self, instance):
```

Comment:
Why do you need a serializer in this case? Is it just to have the data go through the serialization process?

Reply:
Yeah, that's basically it.